packages/server/modules/core/domain/commits/operations.ts

```diff
@@ -86,3 +91,23 @@ export type InsertStreamCommits = (
     trx: Knex.Transaction
   }>
 ) => Promise<number[]>
+
+export type UpdateCommitAndNotify = (
```

Comment:
strictly speaking, `userId` lives in params since it's not optional? but I say we do it later since it matches existing callers

Reply:
@cdriesler generally we want to avoid refactoring anything in these PRs, unless if there's a lot of value for minimum effort (e.g. deleting old code or typescriptifying)